src/OmniSharp.Abstractions/Models/v1/TypeLookup/DocumentationComment.cs

```diff
@@ -30,8 +30,22 @@ private DocumentationComment(string summaryText, DocumentationItem[] typeParamEl
             Exception = exception;
         }
 
+        private DocumentationComment()
```

Comment:
You could make a `this()` constructor call and call your existing constructor with the default values (to avoid having to write the assignments over and over again).
Alternatively, you could remove this constructor and make all the parameters optional in the one above this.